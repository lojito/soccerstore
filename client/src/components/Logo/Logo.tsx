import { type FC } from "react";
import { styled } from "styled-components";

const DivWrapper = styled.div`
  width: 62px;
  display: inline-block;
  padding-top: 5px;
`;

const Logo: FC = () => {
  return (
    <DivWrapper>
      <img src="images/logo.jpg" alt="logo" />
    </DivWrapper>
  );
};

export default Logo;
