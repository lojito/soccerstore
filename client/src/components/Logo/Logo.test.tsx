import { render, screen } from "@testing-library/react";
import Logo from "./Logo";

describe("Logo", () => {
  it("should render correctly", () => {
    const { asFragment } = render(<Logo />);

    expect(asFragment()).toMatchSnapshot();
  });

  it("should have the correct prop", () => {
    render(<Logo />);

    const image = screen.getByRole("img", { name: "logo" });

    expect(image).toHaveAttribute("src", "images/logo.jpg");
  });
});
