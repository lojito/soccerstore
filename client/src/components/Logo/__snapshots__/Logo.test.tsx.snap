// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Logo > should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-beySPh epnVfq"
  >
    <img
      alt="logo"
      src="images/logo.jpg"
    />
  </div>
</DocumentFragment>
`;
